packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -295,6 +296,8 @@ abstract class WidgetsBinding extends BindingBase with SchedulerBinding, Gesture
           return _forceRebuild();
         }
     );
+
+    WidgetInspectorService.instance.initServiceExtensions();
```

Comment:
Seems like if we're doing this now, the previous one (debugWidgetInspector) might as well go there too?

Reply:
I've added 
ext.flutter.inspector.show which is identical to 
ext.flutter.debugWidgetInspector 
I can't remove debugWidgetInspector until users are updated to a version of the IntelliJ plugin that supports it. I've updated flutter tool.